src/Chainweb/Pact/PactService.hs

```diff
@@ -322,6 +329,50 @@ finalizeCheckpointer finalize = do
 _liftCPErr :: Either String a -> PactServiceM cas a
 _liftCPErr = either internalError' return
 
+-- | Read row from coin-table defined in coin contract, retrieving balance and keyset
+-- associated with account name
+--
+readCoinAccount
+    :: PactDbEnv'
+      -- ^ pact db backend (sqlite)
+    -> Text
+      -- ^ account name
+    -> IO (Maybe (T2 Decimal (P.Guard (P.Term P.Name))))
+readCoinAccount (PactDbEnv' (P.PactDbEnv pdb pdbv)) a = rows >>= \case
+    Nothing -> return Nothfing
```

Comment:
`Nothfing` -- but the compiler will catch this for you :)

Reply:
fat fingered on save probably :)